### PR TITLE
Added some fixes

### PR DESCRIPTION
**Pull request by a3ryk**
Perhaps some fixes for this err. 
```js
Sunday, November 28, 2021 1:49 AM LOG [ERROR] Cluster 0 returned an error: TypeError: Cannot read properties of undefined (reading 'users')
    at Function.run (E:\Open sources\ChinoKafuu\src\structures\command\CommandRunner.js:20:44)
    at MessageCreateListener.on (E:\Open sources\ChinoKafuu\src\listeners\commands\MessageCreateListener.js:12:25)
    at E:\Open sources\ChinoKafuu\src\structures\events\ListenerRegistry.js:11:101
    at Array.forEach (<anonymous>)
    at ListenerRegistry.onEvent (E:\Open sources\ChinoKafuu\src\structures\events\ListenerRegistry.js:11:84)
    at Bot.<anonymous> (E:\Open sources\ChinoKafuu\src\structures\events\ListenerRegistry.js:23:136)
    at Bot.emit (node:events:390:28)
    at Shard.emit (E:\Open sources\ChinoKafuu\node_modules\eris\lib\gateway\Shard.js:243:22)
    at Shard.wsEvent (E:\Open sources\ChinoKafuu\node_modules\eris\lib\gateway\Shard.js:1935:16)
    at Shard.onPacket (E:\Open sources\ChinoKafuu\node_modules\eris\lib\gateway\Shard.js:403:16)
```

### CODE REVIEW

**Comment by nayvcake** [comment]
I didn't understand the PR, what could I be correcting in this commit?

**Comment by a3ryk** [comment]
Sorry for my English but when I tried running the dev branch locally and while running clusters it didn't work as expected like it doesn't even start the cluster. 
I could see some codes have used `process.env.CLUSTERS` as boolean. Believe env vars are string so I just fixed it the way it should be?

File: src/structures/Bot.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ module.exports = class Bot extends Client {
          *
          * @type {Database}
          */
-    if (this.database !== undefined) {
```
Comment (a3ryk) [inline src/structures/Bot.js:50]: Also uhh? Won't this always be undefined that way?

**Comment by nayvcake** [comment]
> Sorry for my English but when I tried running the dev branch locally and while running clusters it didn't work as expected like it doesn't even start the cluster.
> I could see some codes have used `process.env.CLUSTERS` as boolean. Believe env vars are string so I just fixed it the way it should be?

That's right, well... Chino Kafuu is currently not using the cluster system. Soon there will be a new cluster system to have the best possible communication between clusters. So when using the cluster, some errors may occur.

File: src/structures/Bot.js (hunk 1 above)
Comment (nayvcake) [reply src/structures/Bot.js:50]: I see this... I will remove it in the next commit.

**Comment by DanielaGC** [comment]
Uhhh... Well, can you change to single quotes? Also, it is not necessary use semicolon

**Comment by a3ryk** [comment]
> > Sorry for my English but when I tried running the dev branch locally and while running clusters it didn't work as expected like it doesn't even start the cluster.
> > I could see some codes have used `process.env.CLUSTERS` as boolean. Believe env vars are string so I just fixed it the way it should be?
> 
> That's right, well... Chino Kafuu is currently not using the cluster system. Soon there will be a new cluster system to have the best possible communication between clusters. So when using the cluster, some errors may occur.

Ah, I understand.

**Pull request merged**